docs/querying/functions.md

```diff
@@ -280,8 +279,10 @@ feature inclusive upper boundaries and exclusive lower boundaries for positive
 values, and vice versa for negative values.) Without a precise alignment of
 boundaries, the function uses interpolation to estimate the fraction. With the
 resulting uncertainty, it becomes irrelevant if the boundaries are inclusive or
-exclusive. The interpolation method is the same as the one used for
-`histogram_quantile()`. See there for more details.
+exclusive.
+
+The interpolation method and handling of float samples is the same as the one
+used for `histogram_quantile()`. See there for more details.
```

Comment:
I think we should mention more prominently (closer to the top) that this function handles both classic and native histograms. (See the 1st paragraph of the documentation of the `histogram_quantile` function below.) As discussed before, there should also be some words of caution that providing lower and upper boundaries far away from bucket boundaries will lead to a large margin of error and that a more informed approach for classic histograms is to use a particular bucket series directly, as described in detail in https://prometheus.io/docs/practices/histograms/ .

Reply:
Ok! Updated the docs to the best of my ability!